save-common/src/commonMain/kotlin/org/cqfn/save/core/files/FileUtils.kt

```diff
@@ -98,6 +98,14 @@ fun FileSystem.readLines(path: Path): List<String> = this.read(path) {
     generateSequence { readUtf8Line() }.toList()
 }
 
+/**
+ * @param path a path to a file
+ * @return string from the file
+ */
+fun FileSystem.readFile(path: Path): String = this.read(path) {
```

Comment:
How is this different from `readLines`?

Reply:
returns the entire file as one line

Reply:
Why not just `readLines().joinToString("\n")` then?

Reply:
Wouldn't that be an extra action?
We read the file line by line first. Then we make a list of string, and then glue it again into a one string.

Reply:
yeah, ok